internal/cluster/cluster.go

```diff
@@ -783,7 +783,7 @@ func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThen strfmt
 
 func (m *Manager) GetClusterByKubeKey(key types.NamespacedName) (*common.Cluster, error) {
 	c := &common.Cluster{}
-	if err := m.db.Take(c, "kube_key_name = ? and kube_key_namespace = ?", key.Name, key.Namespace).Error; err != nil {
+	if err := m.db.Preload("Hosts").Take(c, "kube_key_name = ? and kube_key_namespace = ?", key.Name, key.Namespace).Error; err != nil {
```

Comment:
Why was the hosts Preload added?

Reply:
Because i need to count the ready hosts in order to know if i need to start the installation (because of https://github.com/openshift/assisted-service/pull/955/files#diff-068ee2662f5f4ed13f5a7a7f7e19dc2cfe822d8ed9877a8bb84217ea0d9eeac7R72)